<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>638</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Rockbot Editor 1.0.0</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="game_tab">
       <attribute name="title">
        <string>Game Properties</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_16"/>
      </widget>
      <widget class="QWidget" name="stage_tab">
       <attribute name="title">
        <string>Stages</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5"/>
      </widget>
      <widget class="QWidget" name="maps_tab">
       <attribute name="title">
        <string>Maps</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_17"/>
      </widget>
      <widget class="QWidget" name="weapon_edit_tab">
       <attribute name="title">
        <string>Weapons</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_8"/>
      </widget>
      <widget class="QWidget" name="tab_projectiles">
       <attribute name="title">
        <string>Projectiles</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2"/>
      </widget>
      <widget class="QWidget" name="npc_tab">
       <attribute name="title">
        <string>NPCs</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_11"/>
      </widget>
      <widget class="QWidget" name="ai_tab">
       <attribute name="title">
        <string>Artificial Inteligence</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_12"/>
      </widget>
      <widget class="QWidget" name="object_tab">
       <attribute name="title">
        <string>Objects</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_14"/>
      </widget>
      <widget class="QWidget" name="colorcycle_tab">
       <attribute name="title">
        <string>Colorcycle</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_15"/>
      </widget>
      <widget class="QWidget" name="player_tab">
       <attribute name="title">
        <string>Player Edit</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_18"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionEdit_Tileset"/>
   <addaction name="actionEdit_NPC"/>
   <addaction name="actionAdd_Object"/>
   <addaction name="actionSet_Boss"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="editToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNormal_Edit"/>
   <addaction name="actionLock_Edit"/>
   <addaction name="actionEraser"/>
   <addaction name="actionFill"/>
   <addaction name="separator"/>
   <addaction name="actionLink"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>870</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFILE">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSwap_Maps"/>
    <addaction name="actionReset_Map"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuLevel">
     <property name="title">
      <string>Level</string>
     </property>
     <addaction name="actionOne"/>
     <addaction name="actionTwo"/>
     <addaction name="actionThree"/>
    </widget>
    <addaction name="menuLevel"/>
    <addaction name="actionBeaten"/>
   </widget>
   <addaction name="menuFILE"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normalon>:/toolbar_icons/New</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources/icons/icons.qrc">
     <normaloff>:/toolbar_icons/Open</normaloff>
     <normalon>:/toolbar_icons/Save</normalon>:/toolbar_icons/Open</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources/icons/icons.qrc">
     <normaloff>:/toolbar_icons/Save</normaloff>:/toolbar_icons/Save</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionNormal_Edit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/icons/icons.qrc">
     <normaloff>:/toolbar_icons/draw-freehand.png</normaloff>
     <normalon>:/toolbar_icons/draw-freehand.png</normalon>:/toolbar_icons/draw-freehand.png</iconset>
   </property>
   <property name="text">
    <string>Normal Edit</string>
   </property>
  </action>
  <action name="actionLock_Edit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/toolbar_icons/Lock</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Lock Edit</string>
   </property>
  </action>
  <action name="actionEraser">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/toolbar_icons/draw-eraser.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Eraser</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="resources/icons/icons.qrc">
     <normaloff>:/toolbar_icons/list-add.png</normaloff>:/toolbar_icons/list-add.png</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionFill">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/icons/icons.qrc">
     <normaloff>:/toolbar_icons/Fill</normaloff>:/toolbar_icons/Fill</iconset>
   </property>
   <property name="text">
    <string>Fill</string>
   </property>
  </action>
  <action name="actionLink">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/icons/icons.qrc">
     <normaloff>:/toolbar_icons/connect_no.png</normaloff>:/toolbar_icons/connect_no.png</iconset>
   </property>
   <property name="text">
    <string>Link</string>
   </property>
   <property name="toolTip">
    <string>Link Maps</string>
   </property>
  </action>
  <action name="actionStairs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/icons/icons.qrc">
     <normaloff>:/toolbar_icons/stairs.png</normaloff>:/toolbar_icons/stairs.png</iconset>
   </property>
   <property name="text">
    <string>Stairs</string>
   </property>
  </action>
  <action name="actionEdit_NPC">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/icons/icons.qrc">
     <normaloff>:/toolbar_icons/im-user.png</normaloff>:/toolbar_icons/im-user.png</iconset>
   </property>
   <property name="text">
    <string>Edit NPC</string>
   </property>
  </action>
  <action name="actionEdit_Tileset">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/icons/icons.qrc">
     <normaloff>:/toolbar_icons/format-stroke-color.png</normaloff>:/toolbar_icons/format-stroke-color.png</iconset>
   </property>
   <property name="text">
    <string>Edit Tileset</string>
   </property>
  </action>
  <action name="actionLayer_Level">
   <property name="icon">
    <iconset resource="resources/icons/icons.qrc">
     <normaloff>:/toolbar_icons/format-list-ordered.png</normaloff>:/toolbar_icons/format-list-ordered.png</iconset>
   </property>
   <property name="text">
    <string>Layer Level</string>
   </property>
   <property name="toolTip">
    <string>Layer Level</string>
   </property>
  </action>
  <action name="actionOne">
   <property name="text">
    <string>one</string>
   </property>
  </action>
  <action name="actionTwo">
   <property name="text">
    <string>two</string>
   </property>
  </action>
  <action name="actionThree">
   <property name="text">
    <string>three</string>
   </property>
  </action>
  <action name="actionAdd_Object">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/icons/icons.qrc">
     <normaloff>:/toolbar_icons/media-playlist-shuffle.png</normaloff>:/toolbar_icons/media-playlist-shuffle.png</iconset>
   </property>
   <property name="text">
    <string>Add Object</string>
   </property>
   <property name="toolTip">
    <string>Add Object</string>
   </property>
  </action>
  <action name="actionSet_Boss">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/icons/icons.qrc">
     <normaloff>:/toolbar_icons/Add Npc</normaloff>:/toolbar_icons/Add Npc</iconset>
   </property>
   <property name="text">
    <string>Set Boss</string>
   </property>
   <property name="toolTip">
    <string>Set Boss</string>
   </property>
  </action>
  <action name="actionReset_Map">
   <property name="icon">
    <iconset resource="resources/icons/icons.qrc">
     <normaloff>:/toolbar_icons/archive-remove.png</normaloff>:/toolbar_icons/archive-remove.png</iconset>
   </property>
   <property name="text">
    <string>Reset Map</string>
   </property>
   <property name="toolTip">
    <string>Clear all map tiles</string>
   </property>
  </action>
  <action name="actionSwap_Maps">
   <property name="text">
    <string>Swap Stages</string>
   </property>
  </action>
  <action name="actionBeaten">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/icons/icons.qrc">
     <normaloff>:/toolbar_icons/task-recurring.png</normaloff>:/toolbar_icons/task-recurring.png</iconset>
   </property>
   <property name="text">
    <string>Beaten</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources/icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
